toxcore/network.h

```diff
@@ -71,6 +71,17 @@ typedef enum Net_Packet_Type {
     NET_PACKET_ONION_RECV_2         = 0x8d,
     NET_PACKET_ONION_RECV_1         = 0x8e,
 
+    NET_PACKET_FORWARD_REQUEST      = 0x90,
```

Comment:
Maybe too late, but have you considered using msgpack for the new packets?

Reply:
no